15/umbraco-ui-builder/guides/creating-your-first-integration.md

```diff
@@ -2,15 +2,17 @@
 description: Creating your first integration with Umbraco UI Builder.
 ---
 
-# Creating your first integration
+# Creating Your First Umbraco UI Builder Integration
 
-In this guide, you will find the necessary steps needed for a basic implementation using Umbraco UI Builder to manage a single custom database table.
+This guide walks you through a basic implementation of Umbraco UI Builder to manage a custom database table.
 
 ## Setting Up the Database
 
-Out of the box, Umbraco UI Builder works using PetaPoco as the persistence layer, as this is what ships with Umbraco. If you prefer, it is possible to use a custom [Repository](../advanced/repositories.md). However, for getting started, it is expected that you are using this default strategy.
+Umbraco UI Builder uses PetaPoco as its default persistence layer. You can use a custom [Repository](../advanced/repositories.md), but this guide assumes the default approach.
 
-In this section, let's create a `Person` table to store data. Run the following script in SQL Server Management Studio (SSMS) to create the `Person` table.
+In this section, let's create a `Person` table to store data.
```

Comment:
I haven't checked the rest of the article, but I see mostly "you"s, so we should get rid of this "let us" 🙈 

```suggestion
In this section, you will create a `Person` table to store data.
```